Sentry.xcodeproj/xcshareddata/xcschemes/Sentry.xcscheme

```diff
@@ -64,14 +64,11 @@
                <Test
                   Identifier = "SentryFileIOTrackingIntegrationTests/test_DataConsistency_readUrl_disabled()">
                </Test>
-               <Test
-                  Identifier = "SentryFileIOTrackingIntegrationTests/test_DataConsistency_readUrl_disabled()">
-               </Test>
                <Test
                   Identifier = "SentryHttpTransportTests/testFlush_CalledSequentially_BlocksTwice_disabled()">
                </Test>
                <Test
-                  Identifier = "SentryNetworkTrackerIntegrationTests/testGetRequest_SpanCreatedAndBaggageHeaderAdded_disabled()">
```

Comment:
`test_DataConsistency_readUrl_disabled` and `testGetRequest_SpanCreatedAndBaggageHeaderAdded_disabled` are both still named with the `_disabled` suffix, why were they removed from skipped tests?

Reply:
They're still disabled, only duplicate entries got removed. Just cmd-f the test names in the PR and you'll see they're still both there. 

Was also confusing to me at first!